File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
N = int(input())
lst = list(map(int,input().split()))
cnt = 0

for _ in lst:
    factor = []
    for i in range (1,_+1):
        if _%i == 0:
            factor.append(i)
    if len(factor) == 2:
        cnt += 1

print(cnt)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#include <stdio.h>

int main(void){
    int N;
    scanf("%d", &N);

    int cnt=0;

    for(int i=0; i<N; i++){
        int num;
        scanf("%d", &num);
        if (num<2){
            continue;
        }
        int factor = 0;
        for (int j=2; j<num; j++){
            if (num%j == 0){
                factor++;
                break;
            }
        }
        if (factor == 1){
            continue;
        }
        else if (factor == 0){
            cnt++;
        }
    }

    printf("%d", cnt);
    return 0;
}